.github/workflows/rust.yml

```diff
@@ -36,6 +36,7 @@ jobs:
           toolchain: ${{ env.NIGHTLY_RUST_VERSION }}
           components: rustfmt, clippy
           target: wasm32-unknown-unknown
+          rustflags: ${{ contains(matrix.command, 'wasm32-unknown-unknown') && '--cfg getrandom_backend="wasm_js" -D warnings' || '-D warnings' }}
```

Comment:
Is this still required given the same instruction in `ubrn.config.yaml`?

Reply:
ubrn.config.yaml needs these instructions for the intermediate wasm crate. These are two different things.

1. The marked line tells the rust action which flags to use when compiling the ffi crate (idk why the setup-and-cache rust action doesn't pick up the cargo config).

 2. The ubrn config tells ubrn which flags to use when compiling the intermediate wasm crate which will be located under `crypto-ffi/bindings/js/rust_modules`. For this step ubrn will only listen to the ubrn.config.yml or a cargo config file under rust_modules.